tests/src/dot1q_test.cpp

```diff
@@ -82,18 +82,6 @@ TEST_F(Dot1QTest, Id) {
     EXPECT_EQ(3543, dot1.id());
 }
 
-TEST_F(Dot1QTest, QinQ) {
```

Comment:
why did you removed this one?

Reply:
Because the original code relied in the `Dot1Q` frame to represent both the protocols Dot1Q and Dot1AD. This caused issues when the packets  were rendered: the first frame would have PID 0x8100 (Dot1Q) while the second frame would have PID 0x88a8 (Dot1AD). The correct sequence is with these values inverted: https://upload.wikimedia.org/wikipedia/commons/thumb/5/52/TCPIP_802.1ad_DoubleTag.svg/1280px-TCPIP_802.1ad_DoubleTag.svg.png

Reply:
You actually made me realize I missed something. I'm going to make another change.